tests/test_builtins/pl/test_poland_spec.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+from mimesis.builtins.pl import PolandSpecProvider
+from mimesis.enums import Gender
+from mimesis.providers import Datetime
+
+
+def validate_nip(nip: str) -> bool:
+    """Validate NIP.
+
+    :param nip: nip to validate
+    :return: True if nip is valid, False otherwise
+    """
+    nip_digits = list(map(int, nip))
+    args = (6, 5, 7, 2, 3, 4, 5, 6, 7)
+    sum_v = sum(map(lambda x: x[0] * x[1], zip(args, nip_digits)))
+    checksum_digit = sum_v % 11
+    if checksum_digit > 10:
```

Comment:
Yours `nip()` and `validate_nip()` methods aren't compatible :). In nip() you have here 
```
        if checksum_digit > 9:
            checksum_digit = 0
```
That's why tests are failing from time to time.

I would go step further and would not generate NIPs that have checksum == 10. Allegedly in real life that's how it looks like - officials never give such numbers (although mathematically it's possible of course).

Reply:
@pywkm Thank you for review!

Reply:
@pywkm do you mean I can do something like this: 
```python 
if checksum_digit > 9:
      return self.nip()
nip_digits.append(checksum_digit)
return ''.join(str(d) for d in nip_digits)
```
If we recieve "bad" checksum digit then we can recalculate nip again. But I'm not sure if this is good solution.

Reply:
Yup, it's one of, IMO, acceptable ways. Simplest one.